src/main/resources/application.yml

```diff
@@ -2,22 +2,64 @@ info.app:
   name: Hmpps Education Employment Api
   version: 1.0
 
+azure:
+  application-insights:
+    web:
+      enabled: false
+
 spring:
   application:
     name: hmpps-education-employment-api
   codec:
     max-in-memory-size: 10MB
 
+  security:
+    oauth2:
+      client:
+        scope: read:user, user:email
```

Comment:
do you need to specify the scope here?

Reply:
Ah no - that was part of a hacked experiment - will remove